app/workers/nullify_deactivated_subscribers_worker.rb

```diff
@@ -0,0 +1,16 @@
+class NullifyDeactivatedSubscribersWorker
+  include Sidekiq::Worker
```

Comment:
Do we need this in a queue? as it feels like this should be lower priority that some of the things below default priority?

I've same problem with email archiving. We could have a cleanup queue for all these sort of tasks.

Oh another point I just realised is this could go a bit funky with multiple workers trying it.

Reply:
Good point about multiple workers!

I agree another queue would make sense, shall I add one in this PR and you can rebase off that?

Reply:
I've reworked this to put it on a `cleanup` queue, what do you think?

Reply:
That sounds good 👍